hackathon/models.py

```diff
@@ -166,6 +166,9 @@ class HackProjectScoreCategory(models.Model):
                                    on_delete=models.CASCADE,
                                    related_name="hackprojectscorecategories")
     category = models.CharField(default="", max_length=255)
+    # Score Categories can have different score range (e.g. 1-10, 1-15)
+    # this field sets the upper end of the scale
+    highest_score = models.IntegerField(default=10)
```

Comment:
I'd say change it to `max_score` which is probably more fitting. `Highest score` sounds more like something related to a `highscore` to me.

Reply:
Thanks Stefan, obviously min_score` and `max_score` are the most meaningful names, corrected, tested with assertions, all tests passing.